tests/integration/cv_pipeline.cpp

```diff
@@ -35,24 +36,25 @@ const double roll = 3;
 // with an arbitrary image as input
 int main() {
     cv::Mat image = cv::imread(imagePath);
+    if (!image.data) {
+        std::cout << "failed to open testing image from " << imagePath << std::endl;
```

Comment:
Make this self documenting. Add the cmake command to get the image.

Reply:
Added https://github.com/tritonuas/obcpp/blob/4287233b77214752e9b35ad875f1bf74d560a29e/tests/integration/cv_pipeline.cpp#L12-L14